EnglishFilter/sections/foreign.txt

```diff
@@ -8241,6 +8241,8 @@ interia.pl#%#document.cookie = "lltsg=1";
 !---------------------------------------------------------------------
 !------------------------------ Romanian -----------------------------
 !---------------------------------------------------------------------
+! https://github.com/AdguardTeam/AdguardFilters/issues/77506
+@@||imasdk.googleapis.com/js/sdkloader/ima3_dai.js$important,domain=digi24.ro
```

Comment:
I don't need this rule on my end with extension/app.
Add here `@@||pubads.g.doubleclick.net/ssai/event/*/master.m3u8$domain=digi24.ro`

Reply:
I need, otherwsise adguard gets detected. Pardon me, for the delay. I am currently working on a big project.

Reply:
Ok, thanks.